Dockerfile

```diff
@@ -1,17 +1,19 @@
 ARG GOVERSION=1.17
 ARG GOARCH
 FROM golang:${GOVERSION} as builder
+USER 65534
 ARG GOARCH
 ENV GOARCH=${GOARCH}
+ENV GOCACHE=/tmp/.cache
 WORKDIR /go/src/k8s.io/kube-state-metrics/
-COPY . /go/src/k8s.io/kube-state-metrics/
+COPY --chown=65534:65534 . /go/src/k8s.io/kube-state-metrics/
 
 RUN make build-local
 
 FROM gcr.io/distroless/static:latest-${GOARCH}
-COPY --from=builder /go/src/k8s.io/kube-state-metrics/kube-state-metrics /
+USER 65534
 
-USER nobody
+COPY --from=builder --chown=65534:65534 /go/src/k8s.io/kube-state-metrics/output/kube-state-metrics /
```

Comment:
Why does the binary need to be owned by the user executing it?

Reply:
there is not really a need for it, that's right.
will revert that change.